smriprep/workflows/anatomical.py

```diff
@@ -68,10 +72,11 @@ def init_anat_preproc_wf(
                 freesurfer=True,
                 hires=True,
                 longitudinal=False,
-                num_t1w=1,
+                t1w=['T1w.nii.gz'],
```

Comment:
Maybe add to the run-up to `init_anat_preproc_wf`:

```Python
import nibabel as nb
import numpy as np
nb.Nifti1Image([[[1]]], np.eye(4)).to_filename('T1w.nii.gz')
```

Not sure if you can do any fancy fixture to make sure it's in a temporary directory. Or you can just remove it after.

Reply:
we might be able to dust off the `wf_pre_code` option https://nipype.readthedocs.io/en/0.13.1/users/sphinx_ext.html#configuration-options, though i'm not entirely certain how to use it / we'll still need to remove the file